src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx

```diff
@@ -12,12 +12,14 @@ redirects:
   - /docs/accounts/accounts-billing/new-relic-one-pricing-users/new-relic-one-pricing-billing
   - /docs/accounts/accounts-billing/new-relic-one-pricing-users/pricing-billing
   - /docs/accounts/accounts-billing/new-relic-one-pricing-billing
+  - /docs/licenses/license-information/faq/new-relic-one-pricing-plan-frequently-asked-questions
+  - /docs/licenses/license-information/faq/frequently-asked-questions
 ---
 
-New Relic bills on just two factors: ingested data and billable users. In this doc, we'll explain how our pricing model works, what you get for free, and other details. 
+New Relic has two primary billing factors: ingested data and billable users. In this doc, we'll explain how pricing works, what you get for free, and other details. 
```

Comment:
@zuluecho9 This implies that there are some secondary billing factors...

Reply:
unfortunately there are secondary billing factors (various add-on things). so there are two primary, and several secondary